File: Searching/Linear Search/Linear Search.c

#include <stdio.h> 

int linear_search(int arr[], int n, int x) 
{ 
	for (int i = 0; i < n; i++) 
		if (arr[i] == x) 
			return i; 
	return -1; 
} 

int main(void) 
{ 
	int arr[] = { 1, 2, 3, 4, 5 }; 
	int x = 5; 
	int n = sizeof(arr) / sizeof(arr[0]); 
	int index = linear_search(arr, n, x); 
	(index == -1) ? printf("Element is not present in array") 
				: printf("Element is present at index %d", 
							index); 
	return 0; 
} 

File: Searching/Linear Search/Linear search.py


def linear_search(arr, n, x): 

	for i in range (0, n): 
		if (arr[i] == x): 
			return i; 
	return -1; 


arr = [1, 2, 3, 4, 5]; 
x = 5; 
n = len(arr); 
index = linear_search(arr, n, x) 
if(index == -1): 
	print("Element is not present in array") 
else: 
	print("Element is present at index", index); 
